queries.sql

```diff
@@ -7,4 +7,64 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+--setting the species column to unspecified
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+--Update the species column to digimon for all animals that have a name ending in mon
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species = 'NULL' OR species = '';
+SELECT * FROM animals;
+
+COMMIT;
+SELECT * FROM animals;
+
+--delete all records in the animals table
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+
+--Delete all animals born after Jan 1st, 2022.
+BEGIN;
+DELETE FROM animals
+WHERE date_of_birth  > '2022-01-01';
+--
+BEGIN
+SAVEPOINT first_save;
+--
+
+UPDATE animals
+SET weight_kg  = weight_kg  * -1;
+
```

Comment:
- [ ] Good job so far but I think here it would be great if you add the rollback to the savepoint since it is one of the requirement  ✨🤩🔥